src/Sylius/Bundle/PromotionBundle/DependencyInjection/Configuration.php

```diff
@@ -109,7 +110,7 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                         ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
                                         ->scalarNode('repository')->cannotBeEmpty()->end()
                                         ->scalarNode('factory')->defaultValue(Factory::class)->end()
-                                        ->scalarNode('form')->defaultValue(DefaultResourceType::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('form')->defaultValue(CatalogPromotionType::class)->cannotBeEmpty()->end()
```

Comment:
The `form` field the `CatalogPromotionTranslation` configuration should be adjusted as well